subsys/mgmt/mcumgr/smp/src/smp.c

```diff
@@ -203,7 +203,7 @@ static int smp_handle_single_payload(struct smp_streamer *cbuf, const struct smp
 #if defined(CONFIG_MCUMGR_SMP_COMMAND_STATUS_HOOKS)
 		cmd_recv.group = req_hdr->nh_group;
 		cmd_recv.id = req_hdr->nh_id;
-		cmd_recv.err = MGMT_ERR_EOK;
```

Comment:
Shouldn't this be part of another commit? It does not seem to be explained by commit description.

Reply:
It was repurposed because it shouldn't have been there as per `/** #mcumgr_err_t, used in #MGMT_EVT_OP_CMD_DONE */`